renderer/src/assets/data/interfaces.ts

```diff
@@ -96,15 +96,21 @@ export interface BaseType {
     ev?: [min: number, max: number];
     es?: [min: number, max: number];
   };
-  rune?: Array<{
-    categories: ItemCategory[];
-    string: string;
-    values: number[];
-    tradeId?: string[];
-  }>;
+  rune?: {
+    armour?: RuneDetails;
+    weapon?: RuneDetails;
+    caster?: RuneDetails;
+  };
   tags: string[];
 }
 
+export interface RuneDetails {
+  string: string;
+  tradeId: [string] | null;
+  values: number[];
+  categories?: any[]; // this doesn't exist anymore? :/
```

Comment:
should be fine to remove this from the interface, but I left it here now for review incase the data export I created is incomplete somehow.